From 00b912939e12bb87e75ae7f4d8235c13b66b4f4f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Thomas=20Gst=C3=A4dtner?= <thomas@gstaedtner.net>
Date: Thu, 21 Feb 2013 22:48:25 +0100
Subject: [PATCH 2/2] ran autoupdate to fix autoconf warnings

---
 configure.ac | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index e27fd54..1d7561d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,4 +1,4 @@
-AC_INIT(collection, 4.0.0)
+AC_INIT([collection],[4.0.0])
 AC_CONFIG_SRCDIR(src/main.c)
 AC_CONFIG_HEADERS(src/config.h)
 AM_INIT_AUTOMAKE(dist-bzip2)
@@ -37,4 +37,5 @@ PKG_CHECK_MODULES([libcollectdclient], [libcollectdclient],
 		  [with_libcollectdclient="yes"],
 		  [with_libcollectdclient="no"])
 
-AC_OUTPUT(Makefile share/Makefile src/Makefile)
+AC_CONFIG_FILES([Makefile share/Makefile src/Makefile])
+AC_OUTPUT
-- 
1.8.1.2

